pkg/server/server.go

```diff
@@ -15,6 +16,11 @@ import (
 var log = logger.Package()
 
 func Run(ctx context.Context, c services.Config) error {
+	go func() {
+		log.Infof("Starting pprof on :6060")
+		log.Errorf(http.ListenAndServe("localhost:6060", nil).Error())
```

Comment:
pico-nit: The chance of one of us configuring otto8 wrong is basically zero, but if we wanted to be really pedantic, we could check `c.HTTPListenPort` and return an error if it conflicts with the pprof port